code/modules/mob/living/simple_mob/subtypes/vore/rat.dm

```diff
@@ -53,6 +53,28 @@
 	say_list_type = /datum/say_list/rat
 	ai_holder_type = /datum/ai_holder/simple_mob/melee/rat
 
+/mob/living/simple_mob/vore/aggressive/rat/init_vore()
+	..()
+	var/obj/belly/B = vore_selected
+	B.name = "stomach"
+	B.desc = "In a cruel game of cat-and-mouse gone horribly wrong, you struggle to breathe clearly as the giant rat holds your head in its jaws, the rest of its bulk pinning you to the ground. Slimy slurps and its own muffled squeaking fill your senses as it simultaneously tosses its head while backing up. Quickly, ravenously consuming you, bit by bit, packing you down its gullet no matter how you struggle. Passing by its excited heartbeat, your thoroughly slickened head pushes out into its awaiting stomach, a dark and humid hammock eager to accept the rest of you. Soon, those too-warm, plush walls clench and squeeze around you with undeniable need! A need for mere filling, or, perhaps, a proper meal?"
+
+	B.emote_lists[DM_HOLD] = list(
+		"As time passes, the massive rat’s stomach slowly churns and squeezes down around you, packing you into an easier to carry bundle amidst that oddly soothing massage.",
```

Comment:
I'd think easier to carry would be hyphenated.

Reply:
Was confused at first when you mentioned grammar stuff, since I'm usually on-point with it, but that **does** make sense, yeah. Not a big problem, but, better!